implementations/rust/ockam/ockam_transport_core/src/lib.rs

```diff
@@ -2,6 +2,16 @@
 
 pub use error::TransportError;
 
+#[cfg(feature = "alloc")]
+#[macro_use]
+extern crate alloc;
+
 mod error;
 #[cfg(test)]
 mod error_test;
+pub mod tcp;
+
+/// TCP address type constant
+pub const TCP: u8 = 1;
+
+pub(crate) const CLUSTER_NAME: &str = "_internals.transport.tcp";
```

Comment:
Minor note: maybe we want to rename this cluster to `smoltcp` so that shutdowns between the two tcp stacks (if running in parallel) are independent.

Reply:
Done [here](https://github.com/ockam-network/ockam/pull/2213/commits/87cb060fe0e3ee2d5f7e2750ea7e4ac9f378cfb0)!